test/test_ops.py

```diff
@@ -986,6 +986,25 @@ def test_pad(self):
     helper_test_op([(3,3)], lambda x: torch.nn.functional.pad(x, (0,0,3,4), value=1), lambda x: x.pad(((3,4), None), value=1))
     helper_test_op([(3,3)], lambda x: torch.nn.functional.pad(x, (0,0,0,0), value=1), lambda x: x.pad((None, None), value=1))
 
+  def test_pad_reshape(self):
+    a = Tensor.ones((2,)).reshape((1, 2))
```

Comment:
why does it need to only test Tensor.ones? wouldn't random tensor with shape `(1, 2)` work?

Reply:
Oh, it would work actually. I just had to stick with `Tensor.ones` at the beginning to reproduce the exact faulty view mentioned in the example and kept using it afterward. I will update it to use randomly generated tensors from `helper_test_op`.